db/migrate/20250209171552_add_max_weight_to_predicate_sets.rb

```diff
@@ -0,0 +1,5 @@
+class AddMaxWeightToPredicateSets < ActiveRecord::Migration[7.2]
+  def change
+    add_column :predicate_sets, :max_weight, :float, default: 0.0
```

Comment:
Can `max_weight` be `NULL`? If not, then `null: false` would be nice to add.

Reply:
will add